profiles/logical/holo/holo-business-services/default.nix

```
@@ -0,0 +1,173 @@
+# This is a holo profile for holo business services
+# This set's up node js api's created inside holo-business-services
+# These services should be setup in systemd and you can use 
+# systemctl command to start or stop them
+#
+# e.g.
+#     systemctl start hbs-[service-name].service
+#     systemctl start hbs-registration.service
+
+{ config, lib, pkgs, ... }:
+
+with pkgs;
+
+let
+  registrationSocket = "unix:/run/hbs-registration/hbs-registration.sock";
+
+  authSocket = "unix:/run/hbs-auth/hbs-auth.sock";
+  
+  holochainWorkingDir = "/var/lib/holochain-rsm";
+
+  configureHolochainWorkingDir = "/var/lib/configure-holochain";
+
+  settings = import ../../global-settings.nix;
+
+  networks = import ../../holo-networks.nix;
+
+  holoNetwork = networks.selectNetwork config.system.holoNetwork;
+
+  mongoUri = (builtins.readFile /var/lib/membrane-proof-service/mongodb-uri);
+in
+
+{
+  imports = [
+    ../.
+    ../logging.nix
+  ];
+  environment.systemPackages = with holochainAllBinariesWithDeps.hpos; [ hc-state git hpos-update-cli hpos-holochain-client holochain hc kitsune-p2p-tx2-proxy ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  security.acme = {
+    acceptTerms = true;
+    # REVIEW: maybe a dedicated email for Hydra?
+    email = "przemek.klimek@holo.host";
```

```suggestion
    email = "acme@holo.host";
```
I am slowly removing my address from acme config - too many email from letsencrypt :-)